main.py

```diff
@@ -1 +1,24 @@
 #Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+#All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
```

Comment:
This looks great! A couple things we could do- we could make this into a function, to improve readability and let us all it later. We also could choose to store these variables in a dictionary. Here's an example of what it might look like to implement both of those:


``` python

def get_input():
    # create an empty dictionary
    user_input = {}

    # put user input into that dictionary
    noun = input("Choose a noun: ")
    user_input["noun"] = noun

    plural_noun = input("Choose a plural noun: ")
    user_input["plural_noun"] = plural noun

    # etc
  
    return user_input
```